packages/swirl-components/src/components/swirl-option-list-item/swirl-option-list-item.css

```diff
@@ -37,11 +38,33 @@
 
   @media (--from-desktop-without-touch) {
     min-height: 0;
-    padding: var(--s-space-8) var(--s-space-12);
+    padding: var(--s-space-12);
     gap: var(--s-space-8);
   }
 }
 
+.option-list-item--draggable {
+  & .option-list-item__label {
+    padding-right: calc(var(--s-space-16) + 1.5rem);
+
+    @media (--from-desktop-without-touch) {
+      padding-right: calc(var(--s-space-16) + 1rem);
+    }
+  }
+}
+
+.option-list-item--dragging {
+  z-index: 1;
+  border-radius: var(--s-border-radius-s);
+  background-color: var(--s-surface-overlay-pressed);
```

Comment:
That's on purpose. I've discussed this with Lenni. The "dragging" style is only applied when using keyboard controls to drag. Otherwise the default ghost styles are used.

Reply:
ah ok, good to know